src/EPR.Calculator.API/Builder/Summary/CalcResultSummaryBuilder.cs

```diff
@@ -504,18 +688,43 @@ private static void SetHeaders(CalcResultSummary result, List<MaterialDetail> ma
                 ColumnIndex = ResultSummaryHeaderColumnIndex
             };
 
-            result.ProducerDisposalFeesHeader = new CalcResultSummaryHeader
-            {
-                Name = CalcResultSummaryHeaders.OneProducerDisposalFeesWithBadDebtProvision,
-                ColumnIndex = ProducerDisposalFeesHeaderColumnIndex
-            };
+            result.ProducerDisposalFeesHeaders = GetProducerDisposalFeesHeaders();
 
-            var materialsBreakdownHeader = new List<CalcResultSummaryHeader>();
+            result.MaterialBreakdownHeaders = GetMaterialsBreakdownHeader(result, materials);
+
+            result.ColumnHeaders = GetColumnHeaders(materials);
+        }
+
+        private static List<CalcResultSummaryHeader> GetProducerDisposalFeesHeaders()
+        {
+            return [
+                //Section-4 Title headers
```

Comment:
Please delete this comment as it is irrelevant. It is not section-4 title headers.

Reply:
@johnbenjamin-defra - I have updated the comment, to make it relevant - Done